repos/iiif-storage/src/Controller/CollectionController.php

```diff
@@ -86,6 +94,11 @@ public function viewAction()
 
         $this->paginateControls($viewModel, $collectionRepresentation->getTotalResults(), $this->getManifestsPerPage());
 
+        if (!$this->acl->userIsAllowed($this->getAclResource('iiif-collection'), 'view')) {
```

Comment:
This is the current "shorthand" way of checking if a user has a permission. It's leaning on the `AbstractPsr7ActionController` for the `getAclResource` abstraction. Ideally I would like something like:

```php
$this->userIsAllowedOnSite('iiif-collection', 'view');
```
Or something, to make these calls easy to read and hard to get wrong.